cypress/integration/changeresponse.spec.js

```diff
@@ -0,0 +1,91 @@
+
+describe('network stubbing', () => {
+    it('Return boards', () => {
+        cy.intercept('/api/board', { fixture: 'board.json' }).as('fakeBoard');
+        cy.visit('/');
+        cy.wait('@fakeBoard').then(interception => {
+            const response = interception.response;
```

Comment:
There's not much too need to do assignment here, a better format would be: 

```suggestion
            cy.wait('@fakeBoard').then((data) => {
                expect(data.response.body).to.have.length(4)
            });
```
and so on. But this will do to.